styles.css

```diff
@@ -0,0 +1,35 @@
+#navbar {
+  height: 13%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#seperator01 {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 10px;
+  padding-top: 5em;
+}
+
+.headline {
+  margin-left: 2.4rem;
+  display: flex;
+  width: 83%;
+  height: 55%;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#name {
+  padding-top: 5%;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 3.4em;
+}
+
+#message {
+  font-size: 1.5em;
+  font-family: Arial, Helvetica, sans-serif;
+}

```

Comment:
- [ ] If you notice, there are variations in color, font, width, and spacing to the figma design. Kindly ensure that the design is professionally following the figma design.
- Your design :sunglasses: 


![Screenshot from 2023-01-12 04-10-19](https://user-images.githubusercontent.com/99394374/211952038-fda25b4e-c437-41a0-a5dc-4e2f07065fcc.png)

- Figma design :sunglasses: 

![Screenshot from 2023-01-12 04-09-47](https://user-images.githubusercontent.com/99394374/211952086-90e23563-9558-46f9-aa07-2eb0fad3209b.png)